pallets/schemas/src/rpc/src/tests/rpc_mock.rs

```diff
@@ -0,0 +1,64 @@
+use sp_api::{ApiRef, ProvideRuntimeApi};
```

Comment:
Note this is a duplicate of `pallets/messages/src/rpc/src/tests/rpc_mock.rs`, but didn't want to try to extract it separately.